clang-tools-extra/test/clang-tidy/checkers/bugprone/exception-rethrow.cpp

```diff
@@ -0,0 +1,103 @@
+// RUN: %check_clang_tidy %s bugprone-exception-rethrow %t -- -- -fexceptions
+
+struct exception {};
+
+namespace std {
```

Comment:
For the static analyzer we have `system-header-simulator-cxx.h`, maybe clang-tidy would also benefit from a similar header.

Reply:
that's a job for separate change, as std::move is duplicated in multiple test files.